### PR TITLE
Feature EPL-1213: Write a basic guide of RX concepts

### PR DESCRIPTION
**Pull request by sebastianbarrozo**

### CODE REVIEW

**Review by RomanMagnoli — APPROVED** [review]

**Review by RomanMagnoli — APPROVED** [review]

**Review by Gremiger — APPROVED** [review]

File: docs/developer-guide/etendo-rx/getting-started/requirements.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: Requirements
+---
+The requirements are those registered on [Requirements](/software/getting-started/requirements/).
```
Comment (isaiasb-etendo) [inline docs/developer-guide/etendo-rx/getting-started/requirements.md:4]: This link does not exist. Remove it

**Review by isaiasb-etendo — CHANGES_REQUESTED** [review]

**Comment by RomanMagnoli** [comment]
Checkear PR! Es necesario revisarlo o se declina?

**Comment by Gremiger** [comment]
@isaiasb-etendo The push that you did solved your comment? If yes, please approve the PR so Seba can merge it. Otherwise, let us (and @sebastianbarrozo) know so we can fix it. 

This PR is without activity since last week.

**Comment by isaiasb-etendo** [comment]
> @isaiasb-etendo The push that you did solved your comment? If yes, please approve the PR so Seba can merge it. Otherwise, let us (and @sebastianbarrozo) know so we can fix it.
> 
> This PR is without activity since last week.

@Gremiger @sebastianbarrozo  there is still one reference to a page that does not exist

**Pull request closed without merging**